frontend/src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Nav, NavContainer, NavLogo, NavLogoSpan, NavIcon } from './Navbar.styled.js';
+import { AiFillBell } from 'react-icons/ai';
```

Comment:
Jak będziecie robić Ikonki to to najlepiej będzie jak skorzystamy tylko z ikonek z jednego źródła. 
Niestety w 
react-icons
 nie ma ikon z  Material-UI, więc w takiej sytuacji proponuję "Font Awesome" (fa) - jest taką jedną z popularniejszych i często przytaczanych na kursach